run_scripts_testing/ice_shelf_cavity/isomip_plus/Makefile

```
@@ -0,0 +1,11 @@
+all: isomip_3d_melt_test.log
+
+isomip_3d_melt_test.log: isomip_plus_3d_extruded_layered.py
```

```suggestion
isomip_3d_melt_test.log: isomip_plus_3d_extruded_layered.py clean
```
by making it depend on `clean`, it will remove the log _before_ running the script. This is better because if the run fails with an error (nonzero exit) it stops the Makefile and never gets to the `mv` statement. So if you already had a log from a previous, successful run, the subsequent pytest would think everything is fine as it just looks at the old log. In the workflows CI, it probably would stop at the failed make, before getting to pytest - but better to be safe.